File: example/src/androidTest/java/eu/darken/mvpbakery/example/ExampleTestRunner.java
package eu.darken.mvpbakery.example;

import android.app.Application;
import android.content.Context;

import androidx.test.runner.AndroidJUnitRunner;

public class ExampleTestRunner extends AndroidJUnitRunner {

    @Override
    public Application newApplication(ClassLoader cl, String className, Context context) throws IllegalAccessException, ClassNotFoundException, InstantiationException {
        return super.newApplication(cl, ExampleApplicationMock.class.getName(), context);
    }
}

File: example/src/androidTest/java/eu/darken/mvpbakery/example/ExampleTestRunner.kt
package eu.darken.mvpbakery.example

import android.app.Application
import android.content.Context

import androidx.test.runner.AndroidJUnitRunner


class ExampleTestRunner : AndroidJUnitRunner() {
    override fun newApplication(cl: ClassLoader, className: String, context: Context): Application {
        return super.newApplication(cl, ExampleApplicationMock::class.java.name, context)
    }
}